### PR TITLE
Add new subscription for v1alpha2

### PR DESCRIPTION
**Pull request by MichalKalke**
**Description**
Update config.yaml schema to schema to reflect v1alpha2 subscription api

Changes proposed in this pull request:

- new versioning of `config.yaml`
- adjust operator.go to fix the problem with panic when applying the config
- add checking the subscription group version resource
- add subscription for `v1alpha2`
- adjust` synchronise.go`

**Related issue(s)**
https://github.com/kyma-project/cli/issues/1532

### CODE REVIEW

File: function/pkg/operator/operator_test.go [hunk 1]
```diff
@@ -134,21 +149,21 @@ func Test_applyObject(t *testing.T) {
...
 					result.EXPECT().
 						Update(gomock.Any(), gomock.Any(), gomock.Any()).
-						Return(testObj.DeepCopy(), fmt.Errorf("update failed")).
+						Return(nil, fmt.Errorf("update failed")).
 						AnyTimes()
 
 					return result
 				}(),
-				u:      testObj,
+				u:      testObjModifiedSpec,
```
[the remark this answers is not in the document]
Comment (MichalKalke) [reply function/pkg/operator/operator_test.go:162]: Due to the problem with a setter of field `test` (there is no such a setter) I would like it to stay as it is.

[the remark this answers is not in the document]
Comment (anoipm) [reply function/pkg/operator/operator_test.go:162]: It's complicated to change and I accept this solution

**Review by anoipm — APPROVED** [review]

**Review by jeremyharisch — APPROVED** [review]

**Pull request merged**